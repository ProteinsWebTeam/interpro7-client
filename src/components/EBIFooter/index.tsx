import React from 'react';
import Link from 'components/generic/Link';
import cssBinder from 'styles/cssBinder';
import local from './styles.css';
const css = cssBinder(local);

const EBIFooter = () => {
  const year: string = new Date().getFullYear().toString();
  return (
    <div className="vf-footer">
      <div className="vf-footer__inner">
        <p className="vf-footer__notice">
          <Link
            className="vf-footer__link"
            href="//www.ebi.ac.uk/about/our-impact"
          >
            EMBL-EBI is the home for big data in biology.
          </Link>
        </p>
        <p className="vf-footer__notice">
          We help scientists exploit complex information to make discoveries
          that benefit humankind.
        </p>
        <div className="vf-footer__links-group | vf-grid">
          <div className="vf-links">
            <h4 className="vf-links__heading">
              <Link
                className="vf-heading__link"
                href="//www.ebi.ac.uk/services"
              >
                Services
              </Link>
            </h4>
            <ul className="vf-links__list | vf-list">
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/services/data-resources-and-tools"
                  className="vf-list__link"
                >
                  Data resources and tools
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/submission"
                  className="vf-list__link"
                >
                  Data submission
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link href="//www.ebi.ac.uk/support" className="vf-list__link">
                  Support and feedback
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/licencing"
                  className="vf-list__link"
                >
                  Licensing
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/long-term-data-preservation"
                  className="vf-list__link"
                >
                  Long-term data preservation
                </Link>
              </li>
            </ul>
          </div>

          <div className="vf-links">
            <h4 className="vf-links__heading">
              <Link
                className="vf-heading__link"
                href="//www.ebi.ac.uk/research"
              >
                Research
              </Link>
            </h4>
            <ul className="vf-links__list | vf-list">
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/research/publications"
                  className="vf-list__link"
                >
                  Publications
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/research/groups"
                  className="vf-list__link"
                >
                  Research groups
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/research/postdocs"
                  className="vf-list__link"
                >
                  Postdocs
                </Link>{' '}
                and{' '}
                <Link
                  href="//www.ebi.ac.uk/research/eipp"
                  className="vf-list__link"
                >
                  PhDs
                </Link>
              </li>
            </ul>
          </div>

          <div className="vf-links">
            <h4 className="vf-links__heading">
              <Link
                className="vf-heading__link"
                href="//www.ebi.ac.uk/training"
              >
                Training
              </Link>
            </h4>
            <ul className="vf-links__list | vf-list">
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/training/live-events"
                  className="vf-list__link"
                >
                  Live training
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/training/on-demand"
                  className="vf-list__link"
                >
                  On-demand training
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/training/trainer-support"
                  className="vf-list__link"
                >
                  Support for trainers
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/training/contact-us"
                  className="vf-list__link"
                >
                  Contact organisers
                </Link>
              </li>
            </ul>
          </div>

          <div className="vf-links">
            <h4 className="vf-links__heading">
              <Link
                className="vf-heading__link"
                href="//www.ebi.ac.uk/industry"
              >
                Industry
              </Link>
            </h4>
            <ul className="vf-links__list | vf-list">
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/industry/private/members-area/"
                  className="vf-list__link"
                >
                  Members Area
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/industry/contact-us"
                  className="vf-list__link"
                >
                  Contact Industry team
                </Link>
              </li>
            </ul>
          </div>

          <div className="vf-links">
            <h4 className="vf-links__heading">
              <Link className="vf-heading__link" href="//www.ebi.ac.uk/about">
                About
              </Link>
            </h4>
            <ul className="vf-links__list | vf-list">
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/about/contact"
                  className="vf-list__link"
                >
                  Contact us
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/about/events"
                  className="vf-list__link"
                >
                  Events
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/about/jobs"
                  className="vf-list__link"
                >
                  Jobs
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/about/news"
                  className="vf-list__link"
                >
                  News
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link
                  href="//www.ebi.ac.uk/about/people"
                  className="vf-list__link"
                >
                  People and groups
                </Link>
              </li>
              <li className={css('vf-list__item', 'vf-footer__notice')}>
                <Link href="//intranet.ebi.ac.uk" className="vf-list__link">
                  Intranet for staff
                </Link>
              </li>
            </ul>
          </div>
        </div>
        <p className="vf-footer__legal">
          <span className="vf-footer__legal-text">
            <Link
              className="vf-footer__link"
              href="//www.google.co.uk/maps/place/Hinxton,+Saffron+Walden+CB10+1SD/@52.0815334,0.1891518,17z/data=!3m1!4b1!4m5!3m4!1s0x47d87ccbfbd2538b:0x7bbdb4cde2779ff3!8m2!3d52.0800838!4d0.186415"
            >
              EMBL-EBI, Wellcome Genome Campus, Hinxton, Cambridgeshire, CB10
              1SD, UK.
            </Link>
          </span>
          <span className="vf-footer__legal-text">
            <Link className="vf-footer__link" href="tel:00441223494444">
              Tel: +44 (0)1223 49 44 44
            </Link>
          </span>
          <span className="vf-footer__legal-text">
            <Link
              className="vf-footer__link"
              href="//www.ebi.ac.uk/about/contact"
            >
              Full contact details
            </Link>
          </span>
        </p>
        <p className="vf-footer__legal">
          <span className="vf-footer__legal-text">
            Copyright &copy; EMBL {year}
          </span>
          <span className="vf-footer__legal-text">
            EMBL-EBI is part of the{' '}
            <Link className="vf-footer__link" href="//www.embl.org">
              European Molecular Biology Laboratory
            </Link>
          </span>
          <span className="vf-footer__legal-text">
            <Link
              className="vf-footer__link"
              href="//www.ebi.ac.uk/about/terms-of-use"
            >
              Terms of use
            </Link>
          </span>
        </p>
      </div>
    </div>
  );
};

export default EBIFooter;
