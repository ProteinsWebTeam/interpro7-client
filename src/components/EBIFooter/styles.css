.vf-list__item {
  font-size: 16px !important;
}
