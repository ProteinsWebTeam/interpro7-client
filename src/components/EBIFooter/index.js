// @flow
import React, { PureComponent } from 'react';

import Link from 'components/generic/Link';

import { foundationPartial } from 'styles/foundation';

import ebiGlobalStyles from 'ebi-framework/css/ebi-global.css';
import fonts from 'EBI-Icon-fonts/fonts.css';

const f = foundationPartial(ebiGlobalStyles, fonts);

class EBIFooter extends PureComponent /*:: <{}> */ {
  render() {
    return (
      <div id={f('global-footer')} className={f('global-footer')}>
        <nav
          id={f('global-nav-expanded')}
          className={f('global-nav-expanded', 'row')}
        >
          <div className={f('columns', 'small-12')}>
            <h4 className={f('inline-block')}>
              <Link href="//www.ebi.ac.uk" title="EMBL-EBI">
                EMBL-EBI
              </Link>
            </h4>
            <span className={f('small', 'inline-block', 'padding-left-large')}>
              <Link className={f('readmore')} href="//intranet.ebi.ac.uk">
                <span className={f('icon', 'icon-functional')} data-icon="L" />{' '}
                Intranet for staff
              </Link>
            </span>
          </div>
          <div className={f('medium-up-5', 'small-up-2')}>
            <div className={f('column')}>
              <h5 className={f('research')}>
                <Link href="//www.ebi.ac.uk/services">Services</Link>
              </h5>
              <ul>
                <li className={f('first')}>
                  <Link href="//www.ebi.ac.uk/services">By topic</Link>
                </li>
                <li>
                  <Link href="//www.ebi.ac.uk/services/all">By name (A-Z)</Link>
                </li>
                <li className={f('last')}>
                  <Link href="//www.ebi.ac.uk/support">Help &amp; Support</Link>
                </li>
              </ul>
            </div>
            <div className={f('column')}>
              <h5 className={f('research')}>
                <a href="//www.ebi.ac.uk/research">Research</a>
              </h5>
              <ul>
                <li>
                  <Link href="//www.ebi.ac.uk/research/publications">
                    Publications
                  </Link>
                </li>
                <li>
                  <Link href="//www.ebi.ac.uk/research/groups">
                    Research groups
                  </Link>
                </li>
                <li className={f('last')}>
                  <Link href="//www.ebi.ac.uk/research/postdocs">Postdocs</Link>{' '}
                  &amp; <Link href="//www.ebi.ac.uk/research/eipp">PhDs</Link>
                </li>
              </ul>
            </div>
            <div className={f('column')}>
              <h5 className={f('training')}>
                <Link href="//www.ebi.ac.uk/training">Training</Link>
              </h5>
              <ul>
                <li>
                  <Link href="//www.ebi.ac.uk/training/handson">
                    Train at EBI
                  </Link>
                </li>
                <li>
                  <Link href="//www.ebi.ac.uk/training/roadshow">
                    Train outside EBI
                  </Link>
                </li>
                <li>
                  <Link href="//www.ebi.ac.uk/training/online">
                    Train online
                  </Link>
                </li>
                <li className={f('last')}>
                  <Link href="//www.ebi.ac.uk/training/contact-us">
                    Contact organisers
                  </Link>
                </li>
              </ul>
            </div>
            <div className={f('column')}>
              <h5 className={f('industry')}>
                <Link href="//www.ebi.ac.uk/industry">Industry</Link>
              </h5>
              <ul>
                <li>
                  <Link href="//www.ebi.ac.uk/industry/private">
                    Members Area
                  </Link>
                </li>
                <li>
                  <Link href="//www.ebi.ac.uk/industry/workshops">
                    Workshops
                  </Link>
                </li>
                <li>
                  <Link href="//www.ebi.ac.uk/industry/sme-forum">
                    SME Forum
                  </Link>
                </li>
                <li className={f('last')}>
                  <Link href="//www.ebi.ac.uk/industry/contact">
                    Contact Industry programme
                  </Link>
                </li>
              </ul>
            </div>
            <div className={f('column')}>
              <h5 className={f('about')}>
                <Link href="//www.ebi.ac.uk/about">About</Link>
              </h5>

              <ul>
                <li>
                  <Link href="//www.ebi.ac.uk/about/contact">Contact us</Link>
                </li>
                <li>
                  <Link href="//www.ebi.ac.uk/about/events">Events</Link>
                </li>
                <li>
                  <Link
                    href="//www.ebi.ac.uk/about/jobs"
                    title="Jobs, postdocs, PhDs..."
                  >
                    Jobs
                  </Link>
                </li>
                <li className={f('first')}>
                  <Link href="//www.ebi.ac.uk/about/news">News</Link>
                </li>
                <li>
                  <Link href="//www.ebi.ac.uk/about/people">
                    People &amp; groups
                  </Link>
                </li>
              </ul>
            </div>
          </div>
        </nav>

        <section
          id={f('ebi-footer-meta')}
          className={f('ebi-footer-meta', 'row')}
        >
          <div className={f('columns')}>
            <p className={f('address')}>
              EMBL-EBI, Wellcome Genome Campus, Hinxton, Cambridgeshire, CB10
              1SD, UK. +44 (0)1223 49 44 44
            </p>
            <p className={f('legal')}>
              Copyright Â© EMBL-EBI {new Date().getFullYear()} | EMBL-EBI
              is&nbsp;
              <Link href="//www.embl.org/" rel="noopener">
                part of the European Molecular Biology Laboratory
              </Link>{' '}
              |{' '}
              <Link href="//www.ebi.ac.uk/about/terms-of-use">
                Terms of use
              </Link>
            </p>
          </div>
        </section>
      </div>
    );
  }
}

export default EBIFooter;
