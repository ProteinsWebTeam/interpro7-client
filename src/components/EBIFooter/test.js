import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import EBIFooter from '.';

const renderer = new ShallowRenderer();

describe('<EBIFooter />', () => {
  test('should render', () => {
    renderer.render(<EBIFooter />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
