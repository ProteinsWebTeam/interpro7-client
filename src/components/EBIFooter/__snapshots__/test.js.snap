// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EBIFooter /> should render 1`] = `
<div
  className="global-footer"
  id="global-footer"
>
  <nav
    className="global-nav-expanded row"
    id="global-nav-expanded"
  >
    <div
      className="columns small-12"
    >
      <h4
        className="inline-block"
      >
        <Memo(Connect(Link))
          href="//www.ebi.ac.uk"
          title="EMBL-EBI"
        >
          EMBL-EBI
        </Memo(Connect(Link))>
      </h4>
      <span
        className="small inline-block padding-left-large"
      >
        <Memo(Connect(Link))
          className="readmore"
          href="//intranet.ebi.ac.uk"
        >
          <span
            className="icon icon-functional"
            data-icon="L"
          />
           
          Intranet for staff
        </Memo(Connect(Link))>
      </span>
    </div>
    <div
      className="medium-up-5 small-up-2"
    >
      <div
        className="column"
      >
        <h5
          className="research"
        >
          <Memo(Connect(Link))
            href="//www.ebi.ac.uk/services"
          >
            Services
          </Memo(Connect(Link))>
        </h5>
        <ul>
          <li
            className="first"
          >
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/services"
            >
              By topic
            </Memo(Connect(Link))>
          </li>
          <li>
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/services/all"
            >
              By name (A-Z)
            </Memo(Connect(Link))>
          </li>
          <li
            className="last"
          >
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/support"
            >
              Help & Support
            </Memo(Connect(Link))>
          </li>
        </ul>
      </div>
      <div
        className="column"
      >
        <h5
          className="research"
        >
          <a
            href="//www.ebi.ac.uk/research"
          >
            Research
          </a>
        </h5>
        <ul>
          <li>
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/research/publications"
            >
              Publications
            </Memo(Connect(Link))>
          </li>
          <li>
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/research/groups"
            >
              Research groups
            </Memo(Connect(Link))>
          </li>
          <li
            className="last"
          >
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/research/postdocs"
            >
              Postdocs
            </Memo(Connect(Link))>
             
            & 
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/research/eipp"
            >
              PhDs
            </Memo(Connect(Link))>
          </li>
        </ul>
      </div>
      <div
        className="column"
      >
        <h5
          className="training"
        >
          <Memo(Connect(Link))
            href="//www.ebi.ac.uk/training"
          >
            Training
          </Memo(Connect(Link))>
        </h5>
        <ul>
          <li>
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/training/handson"
            >
              Train at EBI
            </Memo(Connect(Link))>
          </li>
          <li>
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/training/roadshow"
            >
              Train outside EBI
            </Memo(Connect(Link))>
          </li>
          <li>
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/training/online"
            >
              Train online
            </Memo(Connect(Link))>
          </li>
          <li
            className="last"
          >
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/training/contact-us"
            >
              Contact organisers
            </Memo(Connect(Link))>
          </li>
        </ul>
      </div>
      <div
        className="column"
      >
        <h5
          className="industry"
        >
          <Memo(Connect(Link))
            href="//www.ebi.ac.uk/industry"
          >
            Industry
          </Memo(Connect(Link))>
        </h5>
        <ul>
          <li>
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/industry/private"
            >
              Members Area
            </Memo(Connect(Link))>
          </li>
          <li>
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/industry/workshops"
            >
              Workshops
            </Memo(Connect(Link))>
          </li>
          <li>
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/industry/sme-forum"
            >
              SME Forum
            </Memo(Connect(Link))>
          </li>
          <li
            className="last"
          >
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/industry/contact"
            >
              Contact Industry programme
            </Memo(Connect(Link))>
          </li>
        </ul>
      </div>
      <div
        className="column"
      >
        <h5
          className="about"
        >
          <Memo(Connect(Link))
            href="//www.ebi.ac.uk/about"
          >
            About
          </Memo(Connect(Link))>
        </h5>
        <ul>
          <li>
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/about/contact"
            >
              Contact us
            </Memo(Connect(Link))>
          </li>
          <li>
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/about/events"
            >
              Events
            </Memo(Connect(Link))>
          </li>
          <li>
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/about/jobs"
              title="Jobs, postdocs, PhDs..."
            >
              Jobs
            </Memo(Connect(Link))>
          </li>
          <li
            className="first"
          >
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/about/news"
            >
              News
            </Memo(Connect(Link))>
          </li>
          <li>
            <Memo(Connect(Link))
              href="//www.ebi.ac.uk/about/people"
            >
              People & groups
            </Memo(Connect(Link))>
          </li>
        </ul>
      </div>
    </div>
  </nav>
  <section
    className="ebi-footer-meta row"
    id="ebi-footer-meta"
  >
    <div
      className="columns"
    >
      <p
        className="address"
      >
        EMBL-EBI, Wellcome Genome Campus, Hinxton, Cambridgeshire, CB10 1SD, UK. +44 (0)1223 49 44 44
      </p>
      <p
        className="legal"
      >
        Copyright © EMBL-EBI 
        2021
         | EMBL-EBI is 
        <Memo(Connect(Link))
          href="//www.embl.org/"
          rel="noopener"
        >
          part of the European Molecular Biology Laboratory
        </Memo(Connect(Link))>
         
        |
         
        <Memo(Connect(Link))
          href="//www.ebi.ac.uk/about/terms-of-use"
        >
          Terms of use
        </Memo(Connect(Link))>
      </p>
    </div>
  </section>
</div>
`;
